sphinx/writers/latex.py

```diff
@@ -29,7 +29,7 @@
     from docutils.utils.roman import toRoman
```

Comment:
should this be behind an `if sys.version ...` check, rather than catching the `ImportError`?

Reply:
No because it's OS-dependent and not really a version dependent. And this is something we have no real control over. It's been years that it worked like that so...

Reply:
> No because it's OS-dependent and not really a version dependent

ah ok

> It's been years that it worked like that 

and it's definitely still the case?